crates/common/tedge_config/src/tedge_toml/tedge_config.rs

```diff
@@ -246,6 +246,12 @@ define_tedge_config! {
             /// Switch using 501-503 (without operation ID) or 504-506 (with operation ID) SmartREST messages for operation status update
             #[tedge_config(example = "true", default(value = true))]
             use_operation_id: bool,
+
+            child: {
+                /// Attach the c8y_IsDevice fragment to child devices
+                #[tedge_config(example = "false", default(value = false))]
+                append_device_fragment: bool,
+            }
```

Comment:
```suggestion
            child_device: {
                /// Attach the c8y_IsDevice fragment to child devices
                #[tedge_config(example = "false", default(value = false))]
                is_device: bool,
            }
```
Combined property name: `tedge.smartrest.child_device.is_device`. I'm not 100% convinced about that name either. But the intention was to convey the effect of the setting rather than what it technically does under-the-hood.

Reply:
This is a bit verbose, but it does following [101 SmartREST message description](https://cumulocity.com/docs/smartrest/mqtt-static-templates/#101)

```
smartrest.child_device.create_with_device_marker
```

Reply:
That name is as direct as it can be. So, totally fine with that. But, if we really want to make it shorter, I might consider dropping the `create_with` prefix and just keep just `smartrest.child_device.device_marker` or even `smartrest.child_device.is_device_marker`.

Reply:
Though I think the `smartrest.child_device.is_device_marker` is lacking the intent of the config, and do we know if the `c8y_IsDevice` fragment is added/removed if the values changes after the device actually exists (I would suspect that it won't)...if so, then "create_with_device_marker" is more descriptive that it will only be added when creating the device.

Reply:
`101` message doesn't override the existing settings. You will just get an error `41,101,Device with serial 'test-54321' already existing`. It means user can control `c8y_IsDevice` only for the device creation time by `101`.

So, I will update this PR with:
```
smartrest.child_device.create_with_device_marker
```

Reply:
updated: 70a4a5dc106aac91d2c66a29d458ddfa428bde0d